src/ws/context.rs

```diff
@@ -133,7 +133,7 @@ where
 {
     /// Write payload
```

Comment:
Please describe in documentation to this method how user should create `Binary` as user should not manually initialize it.
and instead use proper constructor.

Reply:
Docs added 👍

Reply:
thanks!